src/test/java/net/eventsexpress/app/tests/EventTest.java

```diff
@@ -9,13 +9,25 @@
 public class EventTest extends BaseTest {
 
     @Test(priority = 1)
-    public void verifyUpcomingEvents() {
+    public void verifyUpcomingEventsPage() {
         new Header()
                 .login(EMAIL, PASSWORD)
                 .assertUserLoggedIn();
         new LandingPage()
-                .findEvent();
+                .clickOnFindEventButton();
         new EventsPage()
-                .assertUpcomingEvents();
+                .assertEventsPageOpened();
+    }
+
+    @Test(priority = 2)
+    public void verifyUpcomingEventDetails() {
```

Comment:
please add `@Story` and `@Description` allure annotations

Reply:
Fixed